custom/icds_reports/utils/aggregation_helpers/distributed/inactive_awws.py

```diff
@@ -16,12 +16,24 @@ def __init__(self, last_sync):
         self.last_sync = last_sync
 
     def aggregate(self, cursor):
+        delete_extra_record_query = self.delete_extra_record_query()
         missing_location_query = self.missing_location_query()
         aggregation_query, agg_params = self.aggregate_query()
 
+        cursor.execute(delete_extra_record_query)
         cursor.execute(missing_location_query)
         cursor.execute(aggregation_query, agg_params)
 
+    def delete_extra_record_query(self):
+        return """
+            DELETE FROM icds_reports_aggregateinactiveaww
+                WHERE awc_id IN (
```

Comment:
These `IN` queries can be expensive if the argument contains a lot of entries (they can often result in nested loop). I believe that the subquery here is expected to return few ids, so this shouldn't be an issue. Is that correct?

Reply:
Yeah, The other way around i.e `not in` like query was having lot of ids. But this `IN` query is expected to have  few ids